cmd/abci-cli/abci-cli.go

```diff
@@ -394,14 +397,22 @@ var (
 )
 
 func muxOnCommands(cmd *cobra.Command, pArgs []string) error {
-	if len(pArgs) < 2 {
+
+	if len(pArgs) < 1 && cmd.Use != "batch" {
 		return errBadPersistentArgs
 	}
 
-	subCommand, actualArgs := pArgs[1], pArgs[2:]
+	subCommand, actualArgs := pArgs[0], pArgs[1:]
```

Comment:
@zramsay how come we changed this? That'll make it fail for all the other commands as they use persistentArgs. 
For now am going to revert just this slicing part, and instead make persistentArgs for batch too, how does that sound?